spectool/jdspec.d.ts

```diff
@@ -563,6 +563,28 @@ declare namespace jdspec {
          */
         repo: string
 
+        /**
+         * Optional map of service class to a URL pointing to a service specification markdown file
+         */
+        serviceSpecificationsUrls?: {
+            [serviceClass: number]: string
```

Comment:
Would it be much worse to just have a list of URLs and you figure out the service class from the markdown? less redundancy, so less chances for failure

JSON doesn't really have a map from number so it will look like "449414863": "https://..", but then people may want to use "0x1ac986cf": "https://..."

Reply:
devices are stored in json already. https://github.com/microsoft/jacdac/blob/main/devices/microsoft/research/jmaccv2.json